cmd/cluster/aws/create.go

```diff
@@ -181,6 +182,11 @@ func (o *ValidatedCreateOptions) Complete(ctx context.Context, opts *core.Create
 			*into = value
 		}
 	}
+
+	if err := awsutil.ValidateVPCCIDR(o.VPCCIDR); err != nil {
```

Comment:
should this be called inside `validateAWSOptions()` instead?

Reply:
Makes sense, moved.